### PR TITLE
mockolo 1.1.5 (new formula)

### PR DESCRIPTION
**Pull request by tinder-maxwellelliott**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mockolo.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class Mockolo < Formula
+  desc "Efficient Mock Generator for Swift"
+  homepage "https://github.com/uber/mockolo"
+  url "https://github.com/uber/mockolo/archive/1.1.5.tar.gz"
+  sha256 "53c828299311eb315b096e4d687811da49c0a45ab9647535c2879c144b016bbc"
+
+  pour_bottle? do
```
Comment (SMillerDev) [inline Formula/mockolo.rb]: This is not acceptable. Please don't specify a bottle block, the bot will add it.

File: Formula/mockolo.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    reason "The bottle requires a minimum Swift tools version of 5.1.0"
+    satisfy do
+      match_data = `swift --version` =~ /Swift version (\d+)\.(\d+)\.(\d+)/
+      return false unless match_data
+
+      return match_data[1].to_i >= 5 && match_data[2].to_i >= 1 && match_data[3].to_i >= 0
+    end
+  end
+
+  depends_on :xcode
```
Comment (SMillerDev) [inline Formula/mockolo.rb]: Unless it depends on xcode specifically this can be `depends_on "swift"`

File: Formula/mockolo.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    public protocol Foo {
+        var num: Int { get set }
+        func bar(arg: Float) -> String
+    }")
+    system "#{bin}/mockolo", "-srcs", testpath/"testfile.swift", "-d", testpath/"GeneratedMocks.swift"
+    assert_predicate testpath/"GeneratedMocks.swift", :exist?
+    assert_equal mockgen_output.gsub(/\s+/, " "), shell_output("cat #{testpath/"GeneratedMocks.swift"}").gsub(/\s+/, " ").strip
+  end
+
+  def mockgen_output
```
Comment (SMillerDev) [inline Formula/mockolo.rb]: What is this for?

File: Formula/mockolo.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/mockolo.rb]: What specific parts of xcode does it need?

**Comment by elsh** [comment]
@SMillerDev looks like all checks have passed. Is there anything else that needs changing?

**Comment by elsh** [comment]
cc @vitorgalvao

File: Formula/mockolo.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
+class Mockolo < Formula
+  desc "Efficient Mock Generator for Swift"
+  homepage "https://github.com/uber/mockolo"
+  url "https://github.com/uber/mockolo/archive/1.1.5.tar.gz"
+  sha256 "53c828299311eb315b096e4d687811da49c0a45ab9647535c2879c144b016bbc"
+
+  depends_on :xcode => "11.0"
```
Comment (SMillerDev) [inline Formula/mockolo.rb]: Is this a runtime dependency as well?

Comment (tinder-maxwellelliott) [reply Formula/mockolo.rb]: @elsh I believe we only need this to build mockolo correct?

**Comment by SMillerDev** [comment]
Can you squash your commits and have them match the title of the PR?

**Comment by SMillerDev** [comment]
Thanks @tinder-maxwellelliott! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

File: Formula/mockolo.rb (hunk 4 above)
Comment (elsh) [reply Formula/mockolo.rb]: Yes, it's a build dependency

**Pull request closed without merging**